src/css/wizard-navigation-bar.scss

```diff
@@ -0,0 +1,394 @@
+  background-color: if($wz-indicator-state_ == 'hover', darken($color, 5%), $color);
+}
+
+@mixin wz-state-circle-with-background-and-content($color) {
+  @include wz-state-circle-with-background($color);
+  color: black;
+}
+
+
+// Helper mixin to define a step indicator style like 'small', 'large-filled-symbols' etc.
+//
+// Arguments:
+//   $layout - 'horizontal' | 'vertical'
+//   $width  - step indicator width
+//   $height - step indicator height
+//   $border-width - step indicator border width
+//
+// Apart from the arguments, the `define-style` mixin receives a content block,
+// which is applied for .step-indicator elements and is expected to customize indicator representation.
+// The content block is invoked several times, each time with different arguments passed through global variables:
+//
+//   - $wz-indicator-state_ - indicator state: 'default' or 'hover'
+//   - $wz-indicator-color_ - base color for one of possible indicator states (default, current, done etc.)
+//
+// In addition to these variables, the content block can use the following
+// variables passed through the `define-style` mixin:
+//
+//   - $wz-indicator-width_ - equals to $width argument of the `define-style` mixin
+//   - $wz-indicator-height_ - equals to $height argument of the `define-style` mixin
+//   - $wz-indicator-border-width_ - equals to $border-width argument of the `define-style` mixin
+//
+@mixin wz-define-style($layout, $width: $wz-big-dot-height, $height: $wz-big-dot-height, $border-width: $wz-dot-border-width) {
+
+  // Make variables accessible from the content block
+  $wz-indicator-width_: $width !global;
+  $wz-indicator-height_: $height !global;
+  $wz-indicator-border-width_: $border-width !global;
+
+  @if ($layout == 'horizontal') {
+    padding: ($wz-dot-baseline-distance + $height + $wz-text-padding-bottom) 0 $wz-text-padding-bottom 0;
+  }
+  @else if ($layout == 'vertical') {
+    padding: ($wz-distance-between-steps / 2) ($wz-distance-between-steps / 2) ($wz-distance-between-steps / 2) ($wz-distance-between-steps / 2 + $wz-dot-baseline-distance + $height);
+  }
+
+  li {
+    &:not(:last-child):before {
+      @if ($layout == 'horizontal') {
+        @include wz-horizontal-line($width, $height, $wz-color-default);
+      }
+      @else if ($layout == 'vertical') {
+        @include wz-vertical-line($width, $height, $wz-color-default);
+      }
+    }
+
+    @if ($layout == 'vertical') {
+      a {
+        min-height: $height;
+      }
+    }
+
+    .step-indicator {
+      @if ($layout == 'horizontal') {
+        @include wz-circle-position-horizontal($width, $height, $border-width);
+      }
+      @else if ($layout == 'vertical') {
+        @include wz-circle-position-vertical($width, $height, $border-width);
+      }
+
+      $wz-indicator-state_: 'default' !global;
+      $wz-indicator-color_: $wz-color-default !global;
+      @include wz-state-circle($width, $height, $border-width);
+      @content;
```

Comment:
What does `@content` do?

Reply:
When a mixin is defined like

```
@mixin abc() {
  .child {
    @content
  }
}
```

and is invoked as

```
.parent {
  @include abc() {
    background: red;
  }
}
```

it results in

```
.parent {
  .child {
    background: red;
}
```

It is a mechanism of passing a block of css styles ("content block") from the caller to the mixin. In LESS, we used `.state-default` and `.state-hover` mixins overridden before each call to `define-style`.